iac-extensions/terraform/src/test/resources/checks/ClearTextProtocolsCheck/azurerm_spring_cloud_app.tf

```
@@ -0,0 +1,15 @@
+# Noncompliant@+1 {{Omitting https_only enables clear-text traffic. Make sure it is safe here.}}
+resource "azurerm_spring_cloud_app" "noncompliant_https_missing" {
+}
+
+resource "azurerm_spring_cloud_app" "noncompliant_https_disabled" {
+  https_only = false  # Noncompliant {{Make sure allowing clear-text traffic is safe here.}}
+# ^^^^^^^^^^^^^^^^^^
+}
+
+resource "azurerm_spring_cloud_app" "safe_https_value" {
+  https_only = ABRACADABRA
```

```suggestion
  https_only = true
```
Maybe the negated boolean is a better choice.